langwatch/src/prompt-configs/forms/prompt-config-form/PromptConfigForm.tsx

```
@@ -62,59 +61,78 @@ function InnerPromptConfigForm(props: PromptConfigFormProps) {
+        title: "Failed to save version",
+        description: error.message,
+      });
+    });
+  }, [savedConfig, methods, triggerSaveVersion]);
+
+  const hasDemonstrations =
+    Object.values(
+      savedConfig?.latestVersion.configData.demonstrations?.inline?.records ?? {
+        dummy: [],
+      }
+    )[0]?.length ?? 0 > 0;
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)